client/my-sites/themes/theme-showcase.jsx

```diff
@@ -171,8 +223,12 @@ const ThemeShowcase = React.createClass( {
 } );
 
 export default connect(
-	state => ( {
-		queryParams: getQueryParams( state ),
-		themesList: getThemesList( state ),
-	} )
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
```

Comment:
I know we've discussed this on Slack, but I don't think we need this line here, since you should indeed be able to obtain the `siteId` ownProp from the line above (`( state, { siteId } ) => {`) -- it should be available, since it is also used [here](https://github.com/Automattic/wp-calypso/pull/9698/files#diff-8db99e19c82d4c61d27620e72ccced74R191).